src/screens/collection/components/collectible-item.tsx

```diff
@@ -48,9 +50,22 @@ export const CollectibleItem: FC<Props> = memo(({ item, collectionContract, sele
   const quantityByUser = item?.holders?.find(holder => holder.holder_address === selectedPublicKeyHash)?.quantity ?? 0;
 
   const isAbleToList = quantityByUser > listedByUser;
+  const isListed = item.listed !== undefined && item.listed !== 0;
```

Comment:
isDefined

Reply:
mb rename to listedAmount?

Reply:
isDefined doesn't check if we receive 0

Reply:
replace just "item.listed !== undefined"